ext/dba/dba.c

```diff
@@ -57,6 +57,7 @@ ZEND_BEGIN_MODULE_GLOBALS(dba)
 	const char *default_handler;
 	const dba_handler *default_hptr;
 	HashTable connections;
+	int connection_counter;
```

Comment:
```suggestion
	unsigned int connection_counter;
```

Reply:
I am using %d so I think int is correct and even if someone generates an overflow by opening that many db files the negative numbers would still create unique hashes. Why do you think it should be changed to unsigned?

Reply:
Signed integer overflow is undefined behaviour. You can use `%u` for unsigned ints btw.

Reply:
Fair enough, in case someone opens more than 2 billion dba files :-)